gui/build.py

```diff
@@ -80,14 +80,21 @@ def do_bundle(options):
         if os.path.isdir(dirname):
             shutil.rmtree(dirname)
     # create a intaller
+    if IS_WIN:
```

Comment:
Well, we will! But it is for the next patch - for now, the bundle command is only used in windows, so let keep things simple.